backend/src/main/backend/Table.java

```diff
@@ -0,0 +1,100 @@
+                db.mCreateTable = db.mConnection.prepareStatement("CREATE TABLE IF NOT EXISTS tblMessages (row_id SERIAL PRIMARY KEY, subject VARCHAR(50) NOT NULL, content VARCHAR(500) NOT NULL, user_id INTEGER REFERENCES tblUsers)");
+            }
+            else if(name.equals("tblComment")) {
+                db.mCreateTable =  db.mConnection.prepareStatement("CREATE TABLE IF NOT EXISTS tblComment (row_id SERIAL PRIMARY KEY, message_id INTEGER REFERENCES tblMessages, user_id INTEGER REFERENCES tblUsers, content VARCHAR(500) NOT NULL)");
+            }
+            else if(name.equals("tblUsers")) {
+                db.mCreateTable = db.mConnection.prepareStatement("CREATE TABLE IF NOT EXISTS tblUsers (row_id SERIAL PRIMARY KEY, user_name VARCHAR(500) NOT NULL, first_name VARCHAR(500) NOT NULL, last_name VARCHAR(500) NOT NULL, email VARCHAR(500) NOT NULL)");
+            }
+            else if(name.equals("tblLikes")) {
+                db.mCreateTable = db.mConnection.prepareStatement("CREATE TABLE IF NOT EXISTS tblLikes (row_id SERIAL PRIMARY KEY, user_id INTEGER REFERENCES tblUsers, message_id INTEGER REFERENCES tblMessages, content INTEGER)");    
+            }
+
+            db.mCreateTable.execute();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public void drop(Database db) {
+        String statement = "DROP TABLE IF EXISTS " + name + " CASCADE;"; // Added space before CASCADE
+        try {
+            db.mDropTable = db.mConnection.prepareStatement(statement);
+            db.mDropTable.execute();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public int insertRow(Database db, ArrayList<String> content) {
+        StringBuilder statement = new StringBuilder("INSERT INTO " + name + " VALUES (default");
+
+        for (int i = 0; i < content.size(); i++) {
+            statement.append(", ?");
+        }
+        statement.append(")");
+
+        try {
+            db.mInsertRow = db.mConnection.prepareStatement(statement.toString());
+            for (int i = 0; i < content.size(); i++) {
+                db.mInsertRow.setString(i + 1, content.get(i));
+            }
+            return db.mInsertRow.executeUpdate();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return 0;
+    }
+
+    public int updateRow(Database db, int row_id, String content) {
+        String statement = "UPDATE tblMessages SET content = ? WHERE row_id = ?";
+        try {
+            db.mUpdateRow = db.mConnection.prepareStatement(statement);
+            db.mUpdateRow.setString(1, content);
+            db.mUpdateRow.setInt(2, row_id);
+            return db.mUpdateRow.executeUpdate();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return 0;
+    }
+
+    public int deleteRow(Database db, int row_id) {
+        String statement = "DELETE FROM " + name + " WHERE row_id = ?";
```

Comment:
Currently, this statement variable is re-instantiated every time we call `deleteRow`. Could this be constructed elsewhere to simplify the work that needs done within the context of this method?

Reply:
I have completed a different attempt for this and pushed it to the branch. I created a private class variable called deleteStatement and instantiated it once in the class constructor. Is that better?